site/content/en/docs/formats/synthia.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'SYNTHIA'
+linkTitle: 'SYNTHIA'
+description: ''
+weight: 1
+---
+
+## Format specification
+
+The original SYNTHIA dataset is available
+[here](https://synthia-dataset.net).
+
+Supported annotation types:
+- `Masks`
+
+## Import SYNTHIA dataset
+
+A Datumaro project with a SYNTHIA source can be created in the following way:
+
+```bash
+datum create
+datum import --format synthia <path/to/dataset>
+```
+
+It is also possible to import the dataset using Python API:
+
+```python
+from datumaro.components.dataset import Dataset
+
+synthia_dataset = Dataset.import_from('<path/to/dataset>', 'synthia')
+```
+
+SYNTHIA dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+dataset/
+├── GT/
+│   └── LABELS/
+│       ├── Stereo_Left/
+│       │   ├── Omni_B/
+│       │   │   ├── 000000.png
+│       │   │   ├── 000001.png
+│       |   |   └── ...
+│       │   └── ...
+│       └── Stereo_Right/
+│           ├── Omni_B/
+│           │   ├── 000000.png
+│           │   └── 000001.png
+│           |   └── ...
+│           └── ...
+└── RGB/
+    ├── Stereo_Left/
+    │   ├── Omni_B/
+    │   │   ├── 000000.png
+    │   │   ├── 000001.png
+    |   |   └── ...
+    │   └── ...
+    └── Stereo_Right/
+        ├── Omni_B/
+        │   ├── 000000.png
+        │   └── 000001.png
+        |   └── ...
+        └── ...
+```
+
+- RGB folder containing standard RGB images used for training.
+- GT/LABELS folder containing containing png files (one per image).
+Annotations are given in two channels. The first channel contains
+the class of that pixel (see the table below). The second channel
+contains the unique ID of the instance for those objects
+that are dynamic (cars, pedestrians, etc.).
+
+Also present in the original dataset:
```

Comment:
It's not clear what the purpose of this information is. Is this supposed to say that while the original SYNTHIA dataset has this information, Datumaro ignores it?

Reply:
Datumaro ignores this information when importing, as it can be taken from instance segmentation.

Reply:
Does converter produce these files then?